zwieback:
  model: zwieback
  model_type: urdf
  position: [0, 0, 0.3]
  orientation: [0, 0, 0]

